drivers/bbram/Kconfig

```diff
@@ -14,6 +14,7 @@ source "subsys/logging/Kconfig.template.log_config"
 
 config BBRAM_INIT_PRIORITY
 	int "Init priority"
+	default 65 if BBRAM_STM32
```

Comment:
Would you mind adding a comment on this choice (in commit message for instance)

Reply:
Added comment in a file.